src/trie.rs

```diff
@@ -42,7 +42,7 @@ pub trait Trie<D: DB, H: Hasher> {
     /// return value if key exists, None if key not exist, Error if proof is wrong
     fn verify_proof(
         &self,
-        root_hash: Vec<u8>,
+        root_hash: &[u8],
```

Comment:
modify interface is good also is bad, but I think we should go forward.

Reply:
This will released in 3.1.0. If you want to go forward, lock the minor version in 3.0.

Reply:
4.0.0 is suitable